SunbirdAI/noise-sensors-monitoring
# Add creation and editing of device location

This PR adds the creation and editing of the location of a specific device, through the UI

**How to test this PR:**
* Pull this branch
* Run the app: `docker-compose -f docker-compose-web.yml up`
* Creating a location for a device location:
There are two ways to create a device location:
    * Using the `Add location to devices` button on the main `devices list` page (http://localhost:8000/devices/). This option allows you to enter the location details and choose the device from a dropdown list of all devices
    * Using the `Add location` button on a particular `device_detail` page (http://localhost:8000/devices/<uuid>). This option allows you to add the location to the device whose detail page you are currently viewing. This option will only appear on the `device detail` page of a device that has no location attached to it yet
* Editing a device location: this option will only appear on the `device detail` page of a device whose location was already added. The testing procedure is the same as the above feature
